README.md

```diff
@@ -399,7 +399,7 @@ Be sure to replace the **VERSION** key below with the one of the versions shown
 **Gradle**
 ```gradle
 dependencies {
-    compile 'net.dv8tion:JDA:VERSION'
+    implementation 'net.dv8tion:JDA:VERSION'
```

Comment:
This also needs to be changed below. Also, I think it's reasonable to change this to be `implementation("...")` to be compatible with kotlin-dsl. Additionally, we should add a note that older versions of gradle still require `compile` since the majority is using older versions.

```md
´´´gradle
dependencies {
    ...
}
repositories {
    ....
}
´´´
> Note: Older versions of gradle use `compile` instead of `implementation`!
```